hack/kube/overlays/dev/dex-secret.yaml

```diff
@@ -18,7 +18,7 @@ stringData:
           mode: "false"
     web:
       http: 0.0.0.0:5556
-      allowedOrigins: ["http://localhost:3000", "http://localhost:7440"]
+      allowedOrigins: ["http://localhost:8080", "http://localhost:7440"]
```

Comment:
Unrelated to this changes, I missed the addition of the Temporal URL in the other PR (https://github.com/artefactual-sdps/enduro/pull/653/files#diff-6de9eaa13ba8a389401b40b8466d17aa08141bb6989344b35489724f28e1cb62R21). I think Temporal UI uses a BFF (backend for frontend) auth workflow and it may not be needed here.

Reply:
@jraddaoui huh, interesting. I'll give it a try without the Temporal URL.

Reply:
The Temporal UI login works without this, but for some reason I have to login twice.  When I log out, then log back in with the Dex prompt I get an "unauthorized" message.

![image](https://github.com/artefactual-sdps/enduro/assets/626135/7f6f2e64-23b0-4ae1-b2d6-8f53561ea739)

Then if I log in again, it works. :confused:

Reply:
Hmm, when I logout Temporal UI redirects me to:
http://localhost:7440/login?returnUrl=http%3A%2F%2Flocalhost%3A7440%2Fnamespaces%2Fdefault%2Fworkflows

Then when I try to log in again, URL for the above "unauthorized" page is: http://localhost:7440/login?returnUrl=http%3A%2F%2Flocalhost%3A7440%2Fnamespaces%2Fdefault%2Fworkflows&error=unauthorized

If I remove the "returnUrl" from the first URL, then the login works fine :man_shrugging: